resources/styles/components/task-plan/index.less

```diff
@@ -8,3 +8,9 @@
 @import './select-topics';
 @import './chapter-section';
 @import './footer';
+
+.task-plan {
+  &[data-assignment-type=reading] .dialog > .panel-heading { .tutor-plan(header, 'reading'); }    
```

Comment:
Would a http://lesscss.org/features/#detached-rulesets-feature remove the duplicated child selectors?